### ISSUE TITLE
Evaluating a pre-trained model trained on coco on another dataset having less number of classes

### ISSUE THREAD

**Issue by veds12** [xingyizhou/CenterNet#911]
I want to evaluate the model ```ctdet_coco_dla_2x.pth``` which is provided with the code, on another dataset, having just one class (person). However, since, this model has been trained on coco for 80 classes, it naturally gives poor performance when evaluated as it is since the labels for other objects present in the images are not present in the annotations. Is there some way I can test this model on my dataset for just 'person' class? Or will I have to train a new coco model with only the 'person' class and and then run it on my dataset?

An extension to this question is that is there any way I can fine tune this model (```ctdet_coco_dla_2x.pth```) which has been trained on 80 classes, on my dataset with just one class, such that it predicts only that one class after the finetuning?

**Comment by capricornfati** [comment]
Hello,

you may have a look at the FairMOT project. It is a pedestrian tracker based upon CenterNet. You can modify it to use as a detector. CenterTrack is also an alternative to analyze. 

https://github.com/ifzhang/FairMOT
https://github.com/xingyizhou/CenterTrack

**Comment by veds12** [comment]
Thanks @capricornfati! I'll take a look at these
